### PR TITLE
implement new etl-utils pod

### PR DESCRIPTION
**Pull request by ameijer**
## What does this PR change?
adds new etl utils pod

## Does this PR rely on any other PRs?
none

## How does this PR impact users? (This is the kind of thing that goes in release notes!)
Creates a pod that converts legacy Thanos ETL data to federated ETL data (alpha), supported by request only

## What risks are associated with merging this PR? What is required to fully test this PR?
users can write extra to s3 thanos buckets

## How was this PR tested?
tested in dev cluster

### CODE REVIEW

**Comment by AjayTripathy** [comment]
Just chekcing-- this pod only needs to run duing a migration step right?

**Comment by ameijer** [comment]
@AjayTripathy this actually is designed to run continuously, since the use case @jessegoodier describes is for users with many thanos installs that need to be migrated over to fed ETL slowly - and others that may never migrate at all. Once a certain cluster stops using thanos, this tool will stop placing new bingen files in its respective fed ETL directory... but any other clusters still using thanos will have their data processed by this. Eventually, there will be no more thanos data and this can be shut down. 

Full disclosure - I have not simulated a gradual migration like I describe, but the design of this hopefully puts that in sight.

**Comment by AjayTripathy** [comment]
@ameijer but, to clarify, the user can turn off this pod once the thanos -> aggregator migration is complete?

**Comment by ameijer** [comment]
correct - they can flip the 'enabled' switch to false to remove this pod and stop the conversion pipeline

**Comment by chipzoller** [comment]
Since this is a one-time migration activity, I'm wondering if there's maybe a better way/place to do this rather than the Helm chart.

**Comment by ameijer** [comment]
@chipzoller it would only be a migration activity for certain customers. this would be a permanent part of the app for customers using thanos and waterfowl

**Comment by chipzoller** [comment]
Sorry, can you explain a bit more so I understand?

**Comment by jessegoodier** [comment]
This utility could allow for many use cases. The most obvious being allowing users to continually run thanos and not have to update agents to etl-fed. There are benefits to Thanos, notably grafana dashboards.

In this use case, it will always run and is a part of our deployment mode.

It also can run as a one-time conversion utility. Though I can see other use cases for repair/diagnostics/etc.

**Comment by jessegoodier** [comment]
This utility could allow for many use cases. The most obvious being allowing for users to continually run thanos and not have to update agents.
In which case, it will always run and is a part of our deployment mode.

It also can run as a one-time conversion utility. Though I can see other use cases for repair/diagnostics/etc.

**Review by JesseGoodier — APPROVED** [review]
nice work!

**Comment by jessegoodier** [comment]
Feel free to merge and CP when you know the image is ready.

File: cost-analyzer/values.yaml [hunk 1]
```diff
@@ -533,7 +533,15 @@ kubecostModel:
     #    hosts:
     #      - cost-analyzer-model.local
 
-
+etlUtils:
```
Comment (chipzoller) [inline cost-analyzer/values.yaml:537]: Please add some comment here as to what this does and why anyone would use it.

Comment (jessegoodier) [reply cost-analyzer/values.yaml:537]: good call, added.

**Pull request merged**